apm-sniffer/optional-plugins/optional-spring-plugins/optional-spring-cloud/gateway-2.0.x-plugin/src/main/java/org/apache/skywalking/apm/plugin/spring/cloud/gateway/v20x/HttpClientRequestInterceptor.java

```diff
@@ -45,6 +45,16 @@ public void beforeMethod(final EnhancedInstance objInst,
                              final Object[] allArguments,
                              final Class<?>[] argumentsTypes,
                              final MethodInterceptResult result) throws Throwable {
+        
+        /*
+          In this plug-in the HttpClientFinalizerSendInterceptor depend on the NettyRoutingFilterInterceptor
+          When the NettyRoutingFilterInterceptor is not executed, the HttpClientFinalizerSendInterceptor has no meaning to be executed separately
+          and using ContextManager.activeSpan() method will cause NPE if active span does not exist.
```

Comment:
```suggestion
          In this plug-in, the HttpClientFinalizerSendInterceptor depends on the NettyRoutingFilterInterceptor
          When the NettyRoutingFilterInterceptor is not executed, the HttpClientFinalizerSendInterceptor has no meaning to be executed independently,
          and using ContextManager.activeSpan() method would cause NPE as active span does not exist.
```

Please fix other comments accordingly.